ext/tunemygc/tunemygc_ext.c

```diff
@@ -210,9 +299,12 @@ void Init_tunemygc_ext()
     rb_mTunemygc = rb_define_module("TuneMyGc");
     rb_ivar_set(rb_mTunemygc, id_tunemygc_tracepoint, Qnil);
 
-    rb_define_module_function(rb_mTunemygc, "install_gc_tracepoint", tunemygc_install_gc_tracepoint, 0);
+    rb_define_module_function(rb_mTunemygc, "install_gc_tracepoint", tunemygc_install_gc_tracepoint, 1);
```

Comment:
With the https://github.com/Shopify/tunemygc/pull/2#discussion_r133283227 proposal, I'd keep one method, so that `#gc_counters` would always be maintained (vs. the "either or" that this does). Would be useful to be able to dig into GC stuff further on given hosts, while keeping the counters available for tracing.